charts/kubescape-cloud-operator/templates/kollector/clusterrole.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ClusterRole
```

Comment:
I think the kollector listens only to the Kubescape namespace objects, so we should consider using `Role` instead

Reply:
but didn't we want it to send us the size of the cluster, etc... ?

Reply:
I think we need clusterrole for all these watches:
https://github.com/kubescape/kollector/blob/25cee9cf34cdd6e7af1cd0d11dc19fc329062359/watch/namespacewatch.go#L30
https://github.com/kubescape/kollector/blob/25cee9cf34cdd6e7af1cd0d11dc19fc329062359/watch/secretwatch.go#L35
https://github.com/kubescape/kollector/blob/25cee9cf34cdd6e7af1cd0d11dc19fc329062359/watch/cronjobwatch.go#L29
https://github.com/kubescape/kollector/blob/25cee9cf34cdd6e7af1cd0d11dc19fc329062359/watch/servicewatch.go#L33
https://github.com/kubescape/kollector/blob/25cee9cf34cdd6e7af1cd0d11dc19fc329062359/watch/nodewatch.go#L93
https://github.com/kubescape/kollector/blob/25cee9cf34cdd6e7af1cd0d11dc19fc329062359/watch/podwatch.go#L84